ui/app/models/role-ssh.js

```diff
@@ -82,6 +83,10 @@ export default Model.extend({
     helpText:
       'List of domains for which a client can request a certificate (e.g. `example.com`, or `*` to allow all)',
   }),
+  allowedDomainsTemplate: attr('boolean', {
+    helpText:
+      'Specifies that Allowed domains can be set using identity template policies. Non-templated domains are also permitted.',
```

Comment:
Should `Allowed` be capitalized? if so should `domains` too?

Reply:
I noticed this too! This was copypasta'd from the `allowedUsersTemplate` which had similar weird capitalization. I went ahead and fixed the allowedUsersTemplate helpText because in other help texts we capitalize both when it's an attribute name.